src/init.cpp

```diff
@@ -1144,6 +1145,10 @@ bool AppInitParameterInteraction(const ArgsManager& args)
         return InitError(Untranslated("peertimeout cannot be configured with a negative value."));
     }
 
+    if (args.GetArg("-pingtimeout", TIMEOUT_INTERVAL) <= 0) {
+        return InitError(Untranslated("pingtimeout cannot be configured with a negative value."));
```

Comment:
Nit: Is it a bit pedantic to call it a `nonpositive value` here to include zero? 

Although I see it's consistent with `-peertimeout` error so that's probably why :)

Reply:
I've changed to option to be a binary true/false, so this check is no longer required.